### DISCUSSION TITLE
Bank placeholders ON / OFF by default

### DISCUSSION THREAD

**Discussion by ZeiZeiii** [runelite/runelite#16222] [Q&A] [answered]
I hate when I sell junk from my bank and turn placeholders off. Then I forgot to put it back on, and the next time I gear up, I lose all the placeholders for those items and my bank is a mess.

Ability to automatically toggle placeholders ON when closing the bank would solve this problem.

The same solution for default quantity would be nice as well, but definitely not as important.

**Comment by LlemonDuck** [comment, marked as answer]
This is controlled by the server and cannot be modified by RuneLite alone. RuneLite also cannot instruct the server to change the toggle, since that is disallowed by the Jagex Third Party Client Guidelines. 

https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1
